master/internal/api_user.go

```diff
@@ -237,6 +238,9 @@ func (a *apiServer) GetUserByUsername(
 func (a *apiServer) PostUser(
 	ctx context.Context, req *apiv1.PostUserRequest,
 ) (*apiv1.PostUserResponse, error) {
+	if len(a.m.config.InternalConfig.ExternalSessions.LoginURI) > 1 {
```

Comment:
could we have a `func (c* Config) IsCommunityEdition(...` in `internal/config/config.go` as a helper for this somewhat non-obvious condition?